app/components/organen/inactive-orgaan-table.hbs

```diff
@@ -0,0 +1,31 @@
+<AuDataTable
+  @content={{@content}}
+  @isLoading={{@isLoading}}
+  @noDataMessage="Geen bestuursorganen gevonden"
+  @sort={{@sort}}
+  as |t|
+>
+  <t.content as |c|>
+    <c.header>
+      <AuDataTableThSortable
+        @field="naam"
+        @currentSorting={{@sort}}
+        @label="Naam"
+      />
+      <th>Aantal leden</th>
+      <th>Oprichtingsdatum</th>
+      <th>Einddatum</th>
```

Comment:
so this is the reason for having different tables right? the columns are not the same?

Reply:
yes indeed
it might be possible to work with an if clause, or with some general component which both tables extend, but I think this is the best way for now, as we don't now yet how these tables might change.
It might be a good idea to make a story for a future attempt to generalise all tables.